src/ui/qgsprojectpropertiesbase.ui

```diff
@@ -1798,6 +1831,9 @@
                      </item>
                      <item row="1" column="0">
                       <widget class="QToolButton" name="mAddLayerRestrictionButton">
+                       <property name="toolTip">
+                        <string>Add layer to exclude</string>
```

Comment:
I don't know the context, but maybe clearer `Add layer to the exclude list` or `... to the list of excluded layers`

Reply:
You are right. I think it was a mistake.

Reply:
Can you update the PR so we can merge?

Reply:
I will. But I can only do it on Monday.